cli/kata-env.go

```diff
@@ -25,7 +25,7 @@ import (
 //
 // XXX: Increment for every change to the output format
 // (meaning any change to the EnvInfo type).
-const formatVersion = "1.0.15"
+const formatVersion = "1.0.16"
```

Comment:
Thanks - we also need this command to dump the version of the netmon by implementing a function like:

```
func getNetmonInfo(config oci.RuntimeConfig) (NetmonInfo, error)
```